www/docs/src/content/docs/customizing/studiocms-ui/components/button.mdx

```diff
@@ -0,0 +1,183 @@
+        Button
+      </Button>
+    </PreviewCard>
+  </TabItem>
+  <TabItem label="Code">
+    ```astro
+    ---
+    import { Button } from '@studiocms/ui/components';
+    ---
+
+    <Button color="primary">
+      Button
+    </Button>
+    ```
+  </TabItem>
+</Tabs>
+
+### Disabled
+<Tabs>
+  <TabItem label="Preview">
+    <PreviewCard>
+      <Button color='primary' disabled>
+        Button
+      </Button>
+    </PreviewCard>
+  </TabItem>
+  <TabItem label="Code">
+    ```astro
+    ---
+    import { Button } from '@studiocms/ui/components';
+    ---
+
+    <Button color="primary" disabled>
+      Button
+    </Button>
+    ```
+  </TabItem>
+</Tabs>
+
+### Sizes
+<Tabs>
+  <TabItem label="Preview">
+    <PreviewCard>
+      <Button color='primary' size='sm'>
+        Button (sm)
+      </Button>
+      <Button color='primary'>
+        Button (default)
+      </Button>
+      <Button color='primary' size='lg'>
+        Button (lg)
+      </Button>
+    </PreviewCard>
+  </TabItem>
+  <TabItem label="Code">
+    ```astro
+    ---
+    import { Button } from '@studiocms/ui/components';
+    ---
+
+    <Button color='primary' size='sm'>
+      Button (sm)
+    </Button>
+    <Button color='primary'>
+      Button (default)
+    </Button>
+    <Button color='primary' size='lg'>
+      Button (lg)
+    </Button>
+    ```
+  </TabItem>
+</Tabs>
+
+### Colors
+<Tabs>
+  <TabItem label="Preview">
+    <PreviewCard>
+      <Button color='default'>
+        Default
+      </Button>
+      <Button color='primary'>
+        Primary
+      </Button>
+      <Button color='success'>
+        Success
+      </Button>
+      <Button color='warning'>
+        Warning
+      </Button>
+      <Button color='danger'>
+        Danger
+      </Button>
+    </PreviewCard>
+  </TabItem>
+  <TabItem label="Code">
+    ```astro
+    ---
+    import { Button } from '@studiocms/ui/components';
+    ---
+
+    <Button color='default'>
+      Default
+    </Button>
+    <Button color='primary'>
+      Primary
+    </Button>
+    <Button color='success'>
+      Success
+    </Button>
+    <Button color='warning'>
+      Warning
+    </Button>
+    <Button color='danger'>
+      Danger
+    </Button>
+    ```
+  </TabItem>
+</Tabs>
+
+### Variants
+<Tabs>
+  <TabItem label="Preview">
+    <PreviewCard>
+      <Button color='primary'>
+        Default
+      </Button>
+      <Button color='primary' variant="outlined">
+        Outlined
+      </Button>
+      <Button color='primary' variant="flat">
+        Flat
+      </Button>
+    </PreviewCard>
+  </TabItem>
+  <TabItem label="Code">
+    ```astro
+    ---
+    import { Button } from '@studiocms/ui/components';
+    ---
+
+    <Button color='primary'>
+      Default
+    </Button>
+    <Button color='primary' variant="outlined">
+      Outlined
+    </Button>
+    <Button color='primary' variant="flat">
+      Flat
+    </Button>
+    ```
+  </TabItem>
+</Tabs>
+
+### Polymorphic (rendered as anchor tag)
```

Comment:
![image](https://github.com/user-attachments/assets/184f62cd-6115-434d-ac89-8c5d6bd58aca)

There is two `PreviewCard`s instead of a Preview and a Code?

Reply:
Oh shit lmao will fix later